### ISSUE TITLE
[WebAPI] Implement VideoEncoder on Linux

### ISSUE THREAD

**Issue by pepelsbey** [mdn/content#31913]
## Acceptance Criteria

- [ ] The listed features are documented sufficiently on MDN
- [ ] BCD is updated
- [ ] Interactive example and data repos are updated if appropriate
- [ ] The content has been reviewed as needed

### For folks helping with Firefox related documentation

- [ ] Set bugs to `dev-doc-complete`
- [ ] Add entry to Firefox release notes if feature is enabled in release
- [ ] Add entry to Firefox experimental features page if feature is not yet enabled in release

## Related Gecko bugs

- https://bugzilla.mozilla.org/show_bug.cgi?id=1749047

## Other

- [ ] Check content open issues to see if any pertain to the subject matter. If there are any that can be closed because of the work, do so. If there are any that can be fixed relatively quickly because of the knowledge from completing this issue and you have time, feel free to go ahead and fix them.
- [ ] Check if glossary updates are required for the feature you're documenting - whether an existing term needs to be updated or a new term should be added.
- [ ] Check if BCD update means that content pages need to have experimental markup removed or deprecated markup added (front matter tags and macros).

**Comment by hamishwillee** [comment]
Not necessarily taking this, but have started the investigation. Right now it looks like docs are OK and this mostly BCD. See https://bugzilla.mozilla.org/show_bug.cgi?id=1749047

**Comment by hamishwillee** [comment]
- [ ] BCD - TBD - provided by developer
- [ ] Docs
- [ ] Release notes
- [ ] Experimental features
- [ ] dev-doc-complete

**Comment by pepelsbey** [comment]
@hamishwillee, I [asked Paul](https://bugzilla.mozilla.org/show_bug.cgi?id=1749047#c83) regaring the BCD entry for the feature. Is it something that we can maybe do ourselves?

**Comment by hamishwillee** [comment]
@pepelsbey Yes we could - but when is the trigger since we've asked for this? I can do this next week if no response.

**Comment by hamishwillee** [comment]
OK, I added a BCD.

**Comment by pepelsbey** [comment]
Thank you :)
